client/v2/internal/flags/flags.go

```diff
@@ -17,6 +17,7 @@ const (
 	// FlagNoProposal is the flag convert a gov proposal command into a normal command.
 	// This is used to allow user of chains with custom authority to not use gov submit proposals for usual proposal commands.
 	FlagNoProposal = "no-proposal"
+	FlagAux        = "aux"
```

Comment:
Can we have a go doc for this, like for the others

Reply:
seems aux is mainly for tx instead of query cmd, we might no need change for it